internal/testrunner/runners/system/servicedeployer/kubernetes.go

```diff
@@ -132,17 +132,20 @@ func (ksd KubernetesServiceDeployer) installCustomDefinitions() error {
 
 var _ ServiceDeployer = new(KubernetesServiceDeployer)
 
-func findKubernetesDefinitions(definitionsDir string) (string, error) {
-	if _, err := os.Stat(definitionsDir); err != nil {
-		return "", errors.Wrapf(err, "can't read definitions directory (path: %s)", definitionsDir)
+func checkKubernetesDefinition(definitionsPath string) (string, error) {
```

Comment:
what about only returning the error since the function is called `check...`

Reply:
Not great. If the directory has the empty file, we don't want to return the path, so have to return an empty string to avoid installing custom resources.

Reply:
I think I see what you mean. The output of checkKubernetesDefinition has 3 states:
1. kustomization.yaml found, no errors = return (true, nil)
2. empty folder or .empty file found, no errors = return (false, nil)
3. error = same behaviour as before = return (false, nil)

The name of the first return value (now a boolean) can be "found" or something similar.

A better name for the function is then `containsKustomization`

Reply:
Updated to that.